app/controllers/api/restful_controller.rb

```diff
@@ -1,6 +1,8 @@
 class API::RestfulController < ActionController::Base
   snorlax_used_rest!
 
+  before_filter :doorkeeper_authorize!, only: [:create_action, :update_action, :destroy_action]
```

Comment:
I'm skeptical of this; `create_action`, `update_action`, and `destroy_action` aren't actually controller actions, they're called from the `create`, `update`, and `destroy` actions (and looking at this now they should probably be private to help make that more obvious).

We're using this gem: https://github.com/loomio/snorlax to abstract away heaps of the actual controller implementation, so there's even more opinionated magic than regular rails land (wee!)

In short, the controller actions look like this:
```
def create
  build_resource
  create_action
  respond_with_resource
end

def update
  load_resource
  update_action
  respond_with_resource
end
```
(etc)

with create_action, update_action being overriddes for this particular app.